contracts/DocumentStoreWithRevokeReasons.sol

```diff
@@ -0,0 +1,24 @@
+pragma solidity ^0.5.10;
```

Comment:
any way to prevent this to be deployed in mainnet ? (as it's mainly for test purpose for the moment)

Reply:
why would you do that?

Reply:
he said it was mainly for testing purpose, no real use for this iirc

Reply:
Why not in main net? Anyway this repo doesnt deploy to main net automatically..

Reply:
just to prevent anyone to dig this repo and use this non official contract (I dont mind that much)